src/core/layout/qgslayoutitem.h

```diff
@@ -414,16 +414,60 @@ class CORE_EXPORT QgsLayoutItem : public QgsLayoutObject, public QGraphicsRectIt
      */
     void setParentGroup( QgsLayoutItemGroup *group );
 
+    /**
+     * Behavior of item when exporting to layered outputs.
+     * \since QGIS 3.10
+     */
+    enum ExportLayerBehavior
+    {
+      CanGroupWithAnyOtherItem, //!< Item can be placed on a layer with any other item (default behavior)
+      CanGroupWithItemsOfSameType, //!< Item can only be placed on layers with other items of the same type, but multiple items of this type can be grouped together
+      MustPlaceInOwnLayer, //!< Item must be placed in its own individual layer
```

Comment:
Should you edit the export layer behavior of 3D map items to be set to MustPlaceInOwnLayer ?

Reply:
I don't think so -- these become images in the exports, so are easy to manage when in the same layer as other simple item types. I want to keep MustPlaceInOwnLayer only for complex items which have lots of parts in vector exports (like legends), and would be tricky to manage if they were mixed in a layer with other items.

Reply:
K; I was wondering whether it was worth harmonizing 3D vs 2D map items, i.e. if 2D is a layer of its own, 3D should follow that. But there was indeed no reason other than that. No strong feeling about this, perfectly fine for it to stay as is.